libraries/botframework-connector/src/connectorApi/models/index.ts

```diff
@@ -7,7 +7,7 @@ export * from "botframework-schema";
  * An interface representing ConnectorClientOptions.
  */
 export interface ConnectorClientOptions extends ServiceClientOptions {
-  baseUri: string;
+  baseUri?: string;
```

Comment:
@EricDahlvang can you add a comment indicating why this is optional? Specifically around the behavior that takes place inside of the BotFrameworkAdapter that applies the correct `baseUri`.

If someone uses the ConnectorClient without the BFAdapter, the `baseUri` is required.

Reply:
good point.  comment added